### ISSUE TITLE
Write up Fy21-Q4 report

### ISSUE THREAD

**Issue by psakievich** [psakievich/hfm-fsi#4]

**Comment by michaelasprague** [comment]
Milestone Language: Perform IEA Task 29 validation for the NM80 turbine with blade-resolved simulations with flexible blades, fluid-structure interaction (FSI), and turbulent atmospheric inflow. ExaWind development includes verification of the Geometric Conservation Law (GCL) and load/deformation-mapping capabilities in the context of FSI calculations. Examine and document the sensitivity of loads and wakes to FSI effects and (stretch goal) inclusion of tower/hub.

**Comment by michaelasprague** [comment]
- We need validation data ready for comparison (not sure if Ganesh has that yet).
- Completion would probably look like:
(1) Demonstration of simulation with the "full physics"
(2) Comparison of loads between ExaWind simulations with/without FSI
(3) Provide images of wake with/without FSI (detailed analysis not required)
(4) Comparison of loads with FSI against experimental data
(5) I believe the GCL stuff is all taken care of in previous completed milestone.

**Comment by pscrozi** [comment]
Kevin is working on this one, and putting in more slides.

**Comment by pscrozi** [comment]
Kevin has drafted out the skeleton and sent out needs to Ganesh and Nate to fill it out. Someone added the mesh viz. Making progress on this.

**Comment by pscrozi** [comment]
Waiting on validation statement or plot for the NM80. Data for rigid vs FSI. Once that is done, we can finish the report. Ganesh is running the rigid (he restarted the run) to complete the required number of rotations. Ganesh: it is half way through. Just running the second half, so another 4 days.

**Comment by pscrozi** [comment]
Can't seem to get the validation data on this. Ganesh has tried, but not gotten validation data yet. 

Post-processing loads on first half of rigid case.

Link: https://re.public.polimi.it/bitstream/11311/1175075/1/SCHEJ01-21.pdf      Pages 64-69

**Comment by kevmoor** [comment]
Data scraped from fig 5.8 of the linked report and sent to Ganesh

**Comment by pscrozi** [comment]
Kevin available to help Ganesh with post-processing today (10/20).

Ganesh is willing to fill in the conclusions, future work, etc. (10/20). Why milestone was late, what we had to overcome, will this happen again?

**Comment by pscrozi** [comment]
Ganesh: post-processing will happen today (10/23).

**Comment by kevmoor** [comment]
Here is the link again to the report: 

https://nrel-my.sharepoint.com/:p:/r/personal/mspragu2_nrel_gov/_layouts/15/Doc.aspx?sourcedoc=%7BF6C50130-BF5B-4DCD-A39F-8AD9ACF2368C%7D&file=FloatingTurbineHFMS_fy21q4_milestone.pptx&wdLOR=c28DDC886-CE36-BF4A-91B4-7B87F77ADF93&action=edit&mobileredirect=true

**Comment by pscrozi** [comment]
Kevin looked at the conclusions section. Why was this so overdue? Have obstacles been overcome? What obstacles remain? Mike will spend the afternoon / evening polishing and finishing off.
